2019-09-30 09:29:56,056 ERROR 16576 [-/192.168.17.149/-/19ms GET /database/getFinallyFile?controller=&service=0,1] nodejs.ReferenceError: path is not defined
    at contrArr.forEach (D:\AllCode\svntest\app\controller\database.js:58:26)
    at Array.forEach (<anonymous>)
    at databaseController.makefile (D:\AllCode\svntest\app\controller\database.js:56:18)
    at Object.callFn (D:\AllCode\svntest\node_modules\egg-core\lib\utils\index.js:44:21)
    at Object.classControllerMiddleware (D:\AllCode\svntest\node_modules\egg-core\lib\loader\mixin\controller.js:87:20)
    at Object.callFn (D:\AllCode\svntest\node_modules\@eggjs\router\lib\utils.js:12:21)
    at wrappedController (D:\AllCode\svntest\node_modules\@eggjs\router\lib\egg_router.js:322:18)
    at dispatch (D:\AllCode\svntest\node_modules\koa-compose\index.js:44:32)
    at next (D:\AllCode\svntest\node_modules\koa-compose\index.js:45:18)
    at D:\AllCode\svntest\node_modules\@eggjs\router\lib\router.js:190:18
    at dispatch (D:\AllCode\svntest\node_modules\koa-compose\index.js:44:32)
    at D:\AllCode\svntest\node_modules\koa-compose\index.js:36:12
    at dispatch (D:\AllCode\svntest\node_modules\@eggjs\router\lib\router.js:195:33)
    at dispatch (D:\AllCode\svntest\node_modules\koa\node_modules\koa-compose\index.js:42:32)
    at D:\AllCode\svntest\node_modules\egg-development\app\middleware\egg_loader_trace.js:9:56
    at dispatch (D:\AllCode\svntest\node_modules\koa\node_modules\koa-compose\index.js:42:32)
    at i18n (D:\AllCode\svntest\node_modules\egg-i18n\app\middleware\i18n.js:13:12)
    at dispatch (D:\AllCode\svntest\node_modules\koa\node_modules\koa-compose\index.js:42:32)
    at dispatch (D:\AllCode\svntest\node_modules\egg-security\node_modules\koa-compose\index.js:42:32)
    at dta (D:\AllCode\svntest\node_modules\egg-security\lib\middlewares\dta.js:12:12)
    at dispatch (D:\AllCode\svntest\node_modules\egg-security\node_modules\koa-compose\index.js:42:32)
    at xframe (D:\AllCode\svntest\node_modules\egg-security\lib\middlewares\xframe.js:7:11)
headers: {"Access-Control-Allow-Origin":"http://127.0.0.1:8080","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 16576
hostname: SC-201804270139

2019-09-30 09:29:56,066 ERROR 16576 nodejs.ENOENTError: ENOENT: no such file or directory, open 'D:\AllCode\svntest\app\public\template\controller.js)' (uncaughtException throw 1 times on pid:16576)
ENOENTError: ENOENT: no such file or directory, open 'D:\AllCode\svntest\app\public\template\controller.js)' (uncaughtException throw 1 times on pid:16576)
errno: -4058
code: "ENOENT"
syscall: "open"
path: "D:\\AllCode\\svntest\\app\\public\\template\\controller.js)"
name: "ENOENTError"
pid: 16576
hostname: SC-201804270139

2019-09-30 09:29:56,099 ERROR 16576 nodejs.ENOENTError: ENOENT: no such file or directory, open 'D:\AllCode\svntest\app\public\template\service.js)' (uncaughtException throw 2 times on pid:16576)
ENOENTError: ENOENT: no such file or directory, open 'D:\AllCode\svntest\app\public\template\service.js)' (uncaughtException throw 2 times on pid:16576)
errno: -4058
code: "ENOENT"
syscall: "open"
path: "D:\\AllCode\\svntest\\app\\public\\template\\service.js)"
name: "ENOENTError"
pid: 16576
hostname: SC-201804270139

2019-09-30 09:33:37,405 ERROR 6560 [-/192.168.17.149/-/16ms GET /database/getFinallyFile?controller=&service=0,1] nodejs.ReferenceError: path is not defined
    at contrArr.forEach (D:\AllCode\svntest\app\controller\database.js:58:26)
    at Array.forEach (<anonymous>)
    at databaseController.makefile (D:\AllCode\svntest\app\controller\database.js:56:18)
    at Object.callFn (D:\AllCode\svntest\node_modules\egg-core\lib\utils\index.js:44:21)
    at Object.classControllerMiddleware (D:\AllCode\svntest\node_modules\egg-core\lib\loader\mixin\controller.js:87:20)
    at Object.callFn (D:\AllCode\svntest\node_modules\@eggjs\router\lib\utils.js:12:21)
    at wrappedController (D:\AllCode\svntest\node_modules\@eggjs\router\lib\egg_router.js:322:18)
    at dispatch (D:\AllCode\svntest\node_modules\koa-compose\index.js:44:32)
    at next (D:\AllCode\svntest\node_modules\koa-compose\index.js:45:18)
    at D:\AllCode\svntest\node_modules\@eggjs\router\lib\router.js:190:18
    at dispatch (D:\AllCode\svntest\node_modules\koa-compose\index.js:44:32)
    at D:\AllCode\svntest\node_modules\koa-compose\index.js:36:12
    at dispatch (D:\AllCode\svntest\node_modules\@eggjs\router\lib\router.js:195:33)
    at dispatch (D:\AllCode\svntest\node_modules\koa\node_modules\koa-compose\index.js:42:32)
    at D:\AllCode\svntest\node_modules\egg-development\app\middleware\egg_loader_trace.js:9:56
    at dispatch (D:\AllCode\svntest\node_modules\koa\node_modules\koa-compose\index.js:42:32)
    at i18n (D:\AllCode\svntest\node_modules\egg-i18n\app\middleware\i18n.js:13:12)
    at dispatch (D:\AllCode\svntest\node_modules\koa\node_modules\koa-compose\index.js:42:32)
    at dispatch (D:\AllCode\svntest\node_modules\egg-security\node_modules\koa-compose\index.js:42:32)
    at dta (D:\AllCode\svntest\node_modules\egg-security\lib\middlewares\dta.js:12:12)
    at dispatch (D:\AllCode\svntest\node_modules\egg-security\node_modules\koa-compose\index.js:42:32)
    at xframe (D:\AllCode\svntest\node_modules\egg-security\lib\middlewares\xframe.js:7:11)
headers: {"Access-Control-Allow-Origin":"http://127.0.0.1:8080","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 6560
hostname: SC-201804270139

2019-09-30 09:33:37,415 ERROR 6560 nodejs.ENOENTError: ENOENT: no such file or directory, open 'D:\AllCode\svntest\app\public\template\controller.js)' (uncaughtException throw 1 times on pid:6560)
ENOENTError: ENOENT: no such file or directory, open 'D:\AllCode\svntest\app\public\template\controller.js)' (uncaughtException throw 1 times on pid:6560)
errno: -4058
code: "ENOENT"
syscall: "open"
path: "D:\\AllCode\\svntest\\app\\public\\template\\controller.js)"
name: "ENOENTError"
pid: 6560
hostname: SC-201804270139

2019-09-30 09:33:37,438 ERROR 6560 nodejs.ENOENTError: ENOENT: no such file or directory, open 'D:\AllCode\svntest\app\public\template\service.js)' (uncaughtException throw 2 times on pid:6560)
ENOENTError: ENOENT: no such file or directory, open 'D:\AllCode\svntest\app\public\template\service.js)' (uncaughtException throw 2 times on pid:6560)
errno: -4058
code: "ENOENT"
syscall: "open"
path: "D:\\AllCode\\svntest\\app\\public\\template\\service.js)"
name: "ENOENTError"
pid: 6560
hostname: SC-201804270139

2019-09-30 09:34:37,904 ERROR 3220 nodejs.ENOENTError: ENOENT: no such file or directory, open 'D:\AllCode\svntest\app\public\template\controller.js)' (uncaughtException throw 1 times on pid:3220)
ENOENTError: ENOENT: no such file or directory, open 'D:\AllCode\svntest\app\public\template\controller.js)' (uncaughtException throw 1 times on pid:3220)
errno: -4058
code: "ENOENT"
syscall: "open"
path: "D:\\AllCode\\svntest\\app\\public\\template\\controller.js)"
name: "ENOENTError"
pid: 3220
hostname: SC-201804270139

2019-09-30 09:34:37,927 ERROR 3220 nodejs.ENOENTError: ENOENT: no such file or directory, open 'D:\AllCode\svntest\app\public\template\service.js)' (uncaughtException throw 2 times on pid:3220)
ENOENTError: ENOENT: no such file or directory, open 'D:\AllCode\svntest\app\public\template\service.js)' (uncaughtException throw 2 times on pid:3220)
errno: -4058
code: "ENOENT"
syscall: "open"
path: "D:\\AllCode\\svntest\\app\\public\\template\\service.js)"
name: "ENOENTError"
pid: 3220
hostname: SC-201804270139

2019-09-30 09:56:05,767 ERROR 16784 [-/192.168.17.149/-/25ms GET /database/getFinallyFile?controller=&service=0,1] nodejs.ReferenceError: filepath is not defined
    at serviceArr.forEach (D:\AllCode\svntest\app\controller\database.js:67:30)
    at Array.forEach (<anonymous>)
    at databaseController.makefile (D:\AllCode\svntest\app\controller\database.js:65:20)
    at Object.callFn (D:\AllCode\svntest\node_modules\egg-core\lib\utils\index.js:44:21)
    at Object.classControllerMiddleware (D:\AllCode\svntest\node_modules\egg-core\lib\loader\mixin\controller.js:87:20)
    at Object.callFn (D:\AllCode\svntest\node_modules\@eggjs\router\lib\utils.js:12:21)
    at wrappedController (D:\AllCode\svntest\node_modules\@eggjs\router\lib\egg_router.js:322:18)
    at dispatch (D:\AllCode\svntest\node_modules\koa-compose\index.js:44:32)
    at next (D:\AllCode\svntest\node_modules\koa-compose\index.js:45:18)
    at D:\AllCode\svntest\node_modules\@eggjs\router\lib\router.js:190:18
    at dispatch (D:\AllCode\svntest\node_modules\koa-compose\index.js:44:32)
    at D:\AllCode\svntest\node_modules\koa-compose\index.js:36:12
    at dispatch (D:\AllCode\svntest\node_modules\@eggjs\router\lib\router.js:195:33)
    at dispatch (D:\AllCode\svntest\node_modules\koa\node_modules\koa-compose\index.js:42:32)
    at D:\AllCode\svntest\node_modules\egg-development\app\middleware\egg_loader_trace.js:9:56
    at dispatch (D:\AllCode\svntest\node_modules\koa\node_modules\koa-compose\index.js:42:32)
    at i18n (D:\AllCode\svntest\node_modules\egg-i18n\app\middleware\i18n.js:13:12)
    at dispatch (D:\AllCode\svntest\node_modules\koa\node_modules\koa-compose\index.js:42:32)
    at dispatch (D:\AllCode\svntest\node_modules\egg-security\node_modules\koa-compose\index.js:42:32)
    at dta (D:\AllCode\svntest\node_modules\egg-security\lib\middlewares\dta.js:12:12)
    at dispatch (D:\AllCode\svntest\node_modules\egg-security\node_modules\koa-compose\index.js:42:32)
    at xframe (D:\AllCode\svntest\node_modules\egg-security\lib\middlewares\xframe.js:7:11)
headers: {"Access-Control-Allow-Origin":"http://127.0.0.1:8080","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 16784
hostname: SC-201804270139

2019-09-30 09:56:05,784 ERROR 16784 nodejs.ENOENTError: ENOENT: no such file or directory, open 'D:\AllCode\svntest\app\public\template\controller.js)' (uncaughtException throw 1 times on pid:16784)
ENOENTError: ENOENT: no such file or directory, open 'D:\AllCode\svntest\app\public\template\controller.js)' (uncaughtException throw 1 times on pid:16784)
errno: -4058
code: "ENOENT"
syscall: "open"
path: "D:\\AllCode\\svntest\\app\\public\\template\\controller.js)"
name: "ENOENTError"
pid: 16784
hostname: SC-201804270139

2019-09-30 09:56:05,813 ERROR 16784 nodejs.ENOENTError: ENOENT: no such file or directory, open 'D:\AllCode\svntest\app\public\template\service.js)' (uncaughtException throw 2 times on pid:16784)
ENOENTError: ENOENT: no such file or directory, open 'D:\AllCode\svntest\app\public\template\service.js)' (uncaughtException throw 2 times on pid:16784)
errno: -4058
code: "ENOENT"
syscall: "open"
path: "D:\\AllCode\\svntest\\app\\public\\template\\service.js)"
name: "ENOENTError"
pid: 16784
hostname: SC-201804270139

2019-09-30 09:56:44,065 ERROR 5356 [-/192.168.17.149/-/25ms GET /database/getFinallyFile?controller=&service=0,1] nodejs.Error: template not found: D:\AllCode\svntest\app\public\service\0.js
    at createTemplate (D:\AllCode\svntest\node_modules\nunjucks\src\environment.js:290:15)
    at next (D:\AllCode\svntest\node_modules\nunjucks\src\lib.js:282:7)
    at handle (D:\AllCode\svntest\node_modules\nunjucks\src\environment.js:329:11)
    at D:\AllCode\svntest\node_modules\nunjucks\src\environment.js:339:9
    at next (D:\AllCode\svntest\node_modules\nunjucks\src\lib.js:280:7)
    at Object.asyncIter (D:\AllCode\svntest\node_modules\nunjucks\src\lib.js:286:3)
    at Environment.getTemplate (D:\AllCode\svntest\node_modules\nunjucks\src\environment.js:321:9)
    at Environment.render (D:\AllCode\svntest\node_modules\nunjucks\src\environment.js:360:10)
    at Object.render (D:\AllCode\svntest\node_modules\nunjucks\index.js:92:14)
    at serviceArr.forEach (D:\AllCode\svntest\app\controller\database.js:70:22)
    at Array.forEach (<anonymous>)
    at databaseController.makefile (D:\AllCode\svntest\app\controller\database.js:65:20)
    at Object.callFn (D:\AllCode\svntest\node_modules\egg-core\lib\utils\index.js:44:21)
    at Object.classControllerMiddleware (D:\AllCode\svntest\node_modules\egg-core\lib\loader\mixin\controller.js:87:20)
    at Object.callFn (D:\AllCode\svntest\node_modules\@eggjs\router\lib\utils.js:12:21)
    at wrappedController (D:\AllCode\svntest\node_modules\@eggjs\router\lib\egg_router.js:322:18)
    at dispatch (D:\AllCode\svntest\node_modules\koa-compose\index.js:44:32)
    at next (D:\AllCode\svntest\node_modules\koa-compose\index.js:45:18)
    at D:\AllCode\svntest\node_modules\@eggjs\router\lib\router.js:190:18
    at dispatch (D:\AllCode\svntest\node_modules\koa-compose\index.js:44:32)
    at D:\AllCode\svntest\node_modules\koa-compose\index.js:36:12
    at dispatch (D:\AllCode\svntest\node_modules\@eggjs\router\lib\router.js:195:33)
headers: {"Access-Control-Allow-Origin":"http://127.0.0.1:8080","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 5356
hostname: SC-201804270139

2019-09-30 09:56:44,077 ERROR 5356 nodejs.ENOENTError: ENOENT: no such file or directory, open 'D:\AllCode\svntest\app\public\template\controller.js)' (uncaughtException throw 1 times on pid:5356)
ENOENTError: ENOENT: no such file or directory, open 'D:\AllCode\svntest\app\public\template\controller.js)' (uncaughtException throw 1 times on pid:5356)
errno: -4058
code: "ENOENT"
syscall: "open"
path: "D:\\AllCode\\svntest\\app\\public\\template\\controller.js)"
name: "ENOENTError"
pid: 5356
hostname: SC-201804270139

2019-09-30 09:56:44,122 ERROR 5356 nodejs.ENOENTError: ENOENT: no such file or directory, open 'D:\AllCode\svntest\app\public\template\service.js)' (uncaughtException throw 2 times on pid:5356)
ENOENTError: ENOENT: no such file or directory, open 'D:\AllCode\svntest\app\public\template\service.js)' (uncaughtException throw 2 times on pid:5356)
errno: -4058
code: "ENOENT"
syscall: "open"
path: "D:\\AllCode\\svntest\\app\\public\\template\\service.js)"
name: "ENOENTError"
pid: 5356
hostname: SC-201804270139

2019-09-30 09:57:54,383 ERROR 4568 nodejs.ENOENTError: ENOENT: no such file or directory, open 'D:\AllCode\svntest\app\public\template\controller.js)' (uncaughtException throw 1 times on pid:4568)
ENOENTError: ENOENT: no such file or directory, open 'D:\AllCode\svntest\app\public\template\controller.js)' (uncaughtException throw 1 times on pid:4568)
errno: -4058
code: "ENOENT"
syscall: "open"
path: "D:\\AllCode\\svntest\\app\\public\\template\\controller.js)"
name: "ENOENTError"
pid: 4568
hostname: SC-201804270139

2019-09-30 09:57:54,408 ERROR 4568 nodejs.ENOENTError: ENOENT: no such file or directory, open 'D:\AllCode\svntest\app\public\template\service.js)' (uncaughtException throw 2 times on pid:4568)
ENOENTError: ENOENT: no such file or directory, open 'D:\AllCode\svntest\app\public\template\service.js)' (uncaughtException throw 2 times on pid:4568)
errno: -4058
code: "ENOENT"
syscall: "open"
path: "D:\\AllCode\\svntest\\app\\public\\template\\service.js)"
name: "ENOENTError"
pid: 4568
hostname: SC-201804270139

2019-09-30 09:59:54,070 ERROR 15672 nodejs.ENOENTError: ENOENT: no such file or directory, open 'D:\AllCode\svntest\app\public\template\controller.js)' (uncaughtException throw 1 times on pid:15672)
ENOENTError: ENOENT: no such file or directory, open 'D:\AllCode\svntest\app\public\template\controller.js)' (uncaughtException throw 1 times on pid:15672)
errno: -4058
code: "ENOENT"
syscall: "open"
path: "D:\\AllCode\\svntest\\app\\public\\template\\controller.js)"
name: "ENOENTError"
pid: 15672
hostname: SC-201804270139

2019-09-30 09:59:54,089 ERROR 15672 nodejs.ENOENTError: ENOENT: no such file or directory, open 'D:\AllCode\svntest\app\public\template\service.js)' (uncaughtException throw 2 times on pid:15672)
ENOENTError: ENOENT: no such file or directory, open 'D:\AllCode\svntest\app\public\template\service.js)' (uncaughtException throw 2 times on pid:15672)
errno: -4058
code: "ENOENT"
syscall: "open"
path: "D:\\AllCode\\svntest\\app\\public\\template\\service.js)"
name: "ENOENTError"
pid: 15672
hostname: SC-201804270139

2019-09-30 10:02:25,257 ERROR 8568 nodejs.ENOENTError: ENOENT: no such file or directory, open 'D:\AllCode\svntest\app\public\template\controller.js)' (uncaughtException throw 1 times on pid:8568)
ENOENTError: ENOENT: no such file or directory, open 'D:\AllCode\svntest\app\public\template\controller.js)' (uncaughtException throw 1 times on pid:8568)
errno: -4058
code: "ENOENT"
syscall: "open"
path: "D:\\AllCode\\svntest\\app\\public\\template\\controller.js)"
name: "ENOENTError"
pid: 8568
hostname: SC-201804270139

2019-09-30 10:02:25,277 ERROR 8568 nodejs.ENOENTError: ENOENT: no such file or directory, open 'D:\AllCode\svntest\ D:AllCodesvntestapppublic	emplatecontroller.js' (uncaughtException throw 2 times on pid:8568)
ENOENTError: ENOENT: no such file or directory, open 'D:\AllCode\svntest\ D:AllCodesvntestapppublic	emplatecontroller.js' (uncaughtException throw 2 times on pid:8568)
errno: -4058
code: "ENOENT"
syscall: "open"
path: "D:\\AllCode\\svntest\\ D:AllCodesvntestapppublic\templatecontroller.js"
name: "ENOENTError"
pid: 8568
hostname: SC-201804270139

2019-09-30 10:03:59,336 ERROR 6676 nodejs.ENOENTError: ENOENT: no such file or directory, open 'D:\AllCode\svntest\app\public\template\controller.js)' (uncaughtException throw 1 times on pid:6676)
ENOENTError: ENOENT: no such file or directory, open 'D:\AllCode\svntest\app\public\template\controller.js)' (uncaughtException throw 1 times on pid:6676)
errno: -4058
code: "ENOENT"
syscall: "open"
path: "D:\\AllCode\\svntest\\app\\public\\template\\controller.js)"
name: "ENOENTError"
pid: 6676
hostname: SC-201804270139

2019-09-30 10:03:59,349 ERROR 6676 nodejs.ENOENTError: ENOENT: no such file or directory, open 'D:\AllCode\svntest\ D:AllCodesvntestapppublic	emplatecontroller11111.js' (uncaughtException throw 2 times on pid:6676)
ENOENTError: ENOENT: no such file or directory, open 'D:\AllCode\svntest\ D:AllCodesvntestapppublic	emplatecontroller11111.js' (uncaughtException throw 2 times on pid:6676)
errno: -4058
code: "ENOENT"
syscall: "open"
path: "D:\\AllCode\\svntest\\ D:AllCodesvntestapppublic\templatecontroller11111.js"
name: "ENOENTError"
pid: 6676
hostname: SC-201804270139

2019-09-30 10:06:08,280 ERROR 7688 nodejs.ENOENTError: ENOENT: no such file or directory, open 'D:\AllCode\svntest\app\public\template\controller.js)' (uncaughtException throw 1 times on pid:7688)
ENOENTError: ENOENT: no such file or directory, open 'D:\AllCode\svntest\app\public\template\controller.js)' (uncaughtException throw 1 times on pid:7688)
errno: -4058
code: "ENOENT"
syscall: "open"
path: "D:\\AllCode\\svntest\\app\\public\\template\\controller.js)"
name: "ENOENTError"
pid: 7688
hostname: SC-201804270139

2019-09-30 10:06:08,302 ERROR 7688 nodejs.ENOENTError: ENOENT: no such file or directory, open 'D:\AllCode\svntest\app\public\template\service.js)' (uncaughtException throw 2 times on pid:7688)
ENOENTError: ENOENT: no such file or directory, open 'D:\AllCode\svntest\app\public\template\service.js)' (uncaughtException throw 2 times on pid:7688)
errno: -4058
code: "ENOENT"
syscall: "open"
path: "D:\\AllCode\\svntest\\app\\public\\template\\service.js)"
name: "ENOENTError"
pid: 7688
hostname: SC-201804270139

2019-09-30 10:14:12,143 ERROR 6000 [-/192.168.17.149/-/33ms GET /database/getFinallyFile?controller=&service=0,1] nodejs.ReferenceError: output is not defined
    at databaseController.makefile (D:\AllCode\svntest\app\controller\database.js:87:20)
    at Object.callFn (D:\AllCode\svntest\node_modules\egg-core\lib\utils\index.js:44:21)
    at Object.classControllerMiddleware (D:\AllCode\svntest\node_modules\egg-core\lib\loader\mixin\controller.js:87:20)
    at Object.callFn (D:\AllCode\svntest\node_modules\@eggjs\router\lib\utils.js:12:21)
    at wrappedController (D:\AllCode\svntest\node_modules\@eggjs\router\lib\egg_router.js:322:18)
    at dispatch (D:\AllCode\svntest\node_modules\koa-compose\index.js:44:32)
    at next (D:\AllCode\svntest\node_modules\koa-compose\index.js:45:18)
    at D:\AllCode\svntest\node_modules\@eggjs\router\lib\router.js:190:18
    at dispatch (D:\AllCode\svntest\node_modules\koa-compose\index.js:44:32)
    at D:\AllCode\svntest\node_modules\koa-compose\index.js:36:12
    at dispatch (D:\AllCode\svntest\node_modules\@eggjs\router\lib\router.js:195:33)
    at dispatch (D:\AllCode\svntest\node_modules\koa\node_modules\koa-compose\index.js:42:32)
    at D:\AllCode\svntest\node_modules\egg-development\app\middleware\egg_loader_trace.js:9:56
    at dispatch (D:\AllCode\svntest\node_modules\koa\node_modules\koa-compose\index.js:42:32)
    at i18n (D:\AllCode\svntest\node_modules\egg-i18n\app\middleware\i18n.js:13:12)
    at dispatch (D:\AllCode\svntest\node_modules\koa\node_modules\koa-compose\index.js:42:32)
    at dispatch (D:\AllCode\svntest\node_modules\egg-security\node_modules\koa-compose\index.js:42:32)
    at dta (D:\AllCode\svntest\node_modules\egg-security\lib\middlewares\dta.js:12:12)
    at dispatch (D:\AllCode\svntest\node_modules\egg-security\node_modules\koa-compose\index.js:42:32)
    at xframe (D:\AllCode\svntest\node_modules\egg-security\lib\middlewares\xframe.js:7:11)
    at dispatch (D:\AllCode\svntest\node_modules\egg-security\node_modules\koa-compose\index.js:42:32)
    at xssProtection (D:\AllCode\svntest\node_modules\egg-security\lib\middlewares\xssProtection.js:7:11)
headers: {"Access-Control-Allow-Origin":"http://127.0.0.1:8080","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 6000
hostname: SC-201804270139

2019-09-30 10:14:12,157 ERROR 6000 nodejs.ENOENTError: ENOENT: no such file or directory, open 'D:\AllCode\svntest\app\public\template\controller.js)' (uncaughtException throw 1 times on pid:6000)
ENOENTError: ENOENT: no such file or directory, open 'D:\AllCode\svntest\app\public\template\controller.js)' (uncaughtException throw 1 times on pid:6000)
errno: -4058
code: "ENOENT"
syscall: "open"
path: "D:\\AllCode\\svntest\\app\\public\\template\\controller.js)"
name: "ENOENTError"
pid: 6000
hostname: SC-201804270139

2019-09-30 10:14:12,173 ERROR 6000 nodejs.ENOENTError: ENOENT: no such file or directory, open 'D:\AllCode\svntest\app\public\template\service.js)' (uncaughtException throw 2 times on pid:6000)
ENOENTError: ENOENT: no such file or directory, open 'D:\AllCode\svntest\app\public\template\service.js)' (uncaughtException throw 2 times on pid:6000)
errno: -4058
code: "ENOENT"
syscall: "open"
path: "D:\\AllCode\\svntest\\app\\public\\template\\service.js)"
name: "ENOENTError"
pid: 6000
hostname: SC-201804270139

2019-09-30 10:15:21,003 ERROR 1896 nodejs.ENOENTError: ENOENT: no such file or directory, open 'D:\AllCode\svntest\app\public\template\controller.js)' (uncaughtException throw 1 times on pid:1896)
ENOENTError: ENOENT: no such file or directory, open 'D:\AllCode\svntest\app\public\template\controller.js)' (uncaughtException throw 1 times on pid:1896)
errno: -4058
code: "ENOENT"
syscall: "open"
path: "D:\\AllCode\\svntest\\app\\public\\template\\controller.js)"
name: "ENOENTError"
pid: 1896
hostname: SC-201804270139

2019-09-30 10:15:21,046 ERROR 1896 nodejs.ENOENTError: ENOENT: no such file or directory, open 'D:\AllCode\svntest\app\public\template\service.js)' (uncaughtException throw 2 times on pid:1896)
ENOENTError: ENOENT: no such file or directory, open 'D:\AllCode\svntest\app\public\template\service.js)' (uncaughtException throw 2 times on pid:1896)
errno: -4058
code: "ENOENT"
syscall: "open"
path: "D:\\AllCode\\svntest\\app\\public\\template\\service.js)"
name: "ENOENTError"
pid: 1896
hostname: SC-201804270139

2019-09-30 11:01:02,231 ERROR 14196 nodejs.ServerNoResponseError: server 127.0.0.1:52521 no response in 20002ms, maybe the socket is end on the other side. (address: 127.0.0.1:52521)
    at Timeout._heartbeatTimer.setInterval [as _onTimeout] (D:\AllCode\svntest\node_modules\tcp-base\lib\base.js:427:21)
    at ontimeout (timers.js:427:11)
    at tryOnTimeout (timers.js:289:5)
    at listOnTimeout (timers.js:252:5)
    at Timer.processTimers (timers.js:212:10)
name: "ServerNoResponseError"
pid: 14196
hostname: SC-201804270139

2019-09-30 11:08:48,745 ERROR 16036 nodejs.ClusterClientNoResponseError: client no response in 506514ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (D:\AllCode\svntest\node_modules\cluster-client\lib\leader.js:77:23)
    at ontimeout (timers.js:427:11)
    at tryOnTimeout (timers.js:289:5)
    at listOnTimeout (timers.js:252:5)
    at Timer.processTimers (timers.js:212:10)
name: "ClusterClientNoResponseError"
pid: 16036
hostname: SC-201804270139

2019-09-30 11:11:40,836 ERROR 17068 nodejs.ENOENTError: ENOENT: no such file or directory, open 'James' (uncaughtException throw 1 times on pid:17068)
ENOENTError: ENOENT: no such file or directory, open 'James' (uncaughtException throw 1 times on pid:17068)
errno: -4058
code: "ENOENT"
syscall: "open"
path: "James"
name: "ENOENTError"
pid: 17068
hostname: SC-201804270139

2019-09-30 11:11:40,861 ERROR 17068 nodejs.ENOENTError: ENOENT: no such file or directory, open 'James' (uncaughtException throw 2 times on pid:17068)
ENOENTError: ENOENT: no such file or directory, open 'James' (uncaughtException throw 2 times on pid:17068)
errno: -4058
code: "ENOENT"
syscall: "open"
path: "James"
name: "ENOENTError"
pid: 17068
hostname: SC-201804270139

2019-09-30 13:34:15,144 ERROR 9348 nodejs.unhandledExceptionError: read ECONNRESET
    at TCP.onread (net.js:659:25)
errno: "ECONNRESET"
code: "ECONNRESET"
syscall: "read"
name: "unhandledExceptionError"
pid: 9348
hostname: SC-201804270139

2019-09-30 13:38:06,077 ERROR 5336 [-/192.168.17.149/-/1162ms GET /database/find] nodejs.SequelizeConnectionRefusedError: connect ECONNREFUSED 127.0.0.1:3306
    at Promise.tap.then.catch.err (D:\AllCode\svntest\node_modules\sequelize\lib\dialects\mysql\connection-manager.js:123:19)
From previous event:
    at ConnectionManager.connect (D:\AllCode\svntest\node_modules\sequelize\lib\dialects\mysql\connection-manager.js:120:13)
    at sequelize.runHooks.then (D:\AllCode\svntest\node_modules\sequelize\lib\dialects\abstract\connection-manager.js:311:50)
    at runCallback (timers.js:696:18)
    at tryOnImmediate (timers.js:667:5)
    at processImmediate (timers.js:649:5)
From previous event:
    at ConnectionManager._connect (D:\AllCode\svntest\node_modules\sequelize\lib\dialects\abstract\connection-manager.js:311:8)
    at ConnectionManager.getConnection (D:\AllCode\svntest\node_modules\sequelize\lib\dialects\abstract\connection-manager.js:247:46)
    at Promise.try (D:\AllCode\svntest\node_modules\sequelize\lib\sequelize.js:640:36)
From previous event:
    at Promise.resolve.retry (D:\AllCode\svntest\node_modules\sequelize\lib\sequelize.js:631:53)
    at D:\AllCode\svntest\node_modules\retry-as-promised\index.js:70:21
    at new Promise (<anonymous>)
    at retryAsPromised (D:\AllCode\svntest\node_modules\retry-as-promised\index.js:60:10)
    at Promise.try (D:\AllCode\svntest\node_modules\sequelize\lib\sequelize.js:631:30)
    at Sequelize.query (D:\AllCode\svntest\node_modules\sequelize\lib\sequelize.js:580:23)
    at databaseService.find (D:\AllCode\svntest\app\service\database.js:11:45)
    at databaseController.index (D:\AllCode\svntest\app\controller\database.js:32:51)
    at Object.callFn (D:\AllCode\svntest\node_modules\egg-core\lib\utils\index.js:44:21)
    at Object.classControllerMiddleware (D:\AllCode\svntest\node_modules\egg-core\lib\loader\mixin\controller.js:87:20)
    at Object.callFn (D:\AllCode\svntest\node_modules\@eggjs\router\lib\utils.js:12:21)
    at wrappedController (D:\AllCode\svntest\node_modules\@eggjs\router\lib\egg_router.js:322:18)
    at dispatch (D:\AllCode\svntest\node_modules\koa-compose\index.js:44:32)
    at next (D:\AllCode\svntest\node_modules\koa-compose\index.js:45:18)
    at D:\AllCode\svntest\node_modules\@eggjs\router\lib\router.js:190:18
    at dispatch (D:\AllCode\svntest\node_modules\koa-compose\index.js:44:32)
    at D:\AllCode\svntest\node_modules\koa-compose\index.js:36:12
    at dispatch (D:\AllCode\svntest\node_modules\@eggjs\router\lib\router.js:195:33)
    at dispatch (D:\AllCode\svntest\node_modules\koa\node_modules\koa-compose\index.js:42:32)
    at D:\AllCode\svntest\node_modules\egg-development\app\middleware\egg_loader_trace.js:9:56
    at dispatch (D:\AllCode\svntest\node_modules\koa\node_modules\koa-compose\index.js:42:32)
    at i18n (D:\AllCode\svntest\node_modules\egg-i18n\app\middleware\i18n.js:13:12)
    at dispatch (D:\AllCode\svntest\node_modules\koa\node_modules\koa-compose\index.js:42:32)
From previous event:
    at Sequelize.query (D:\AllCode\svntest\node_modules\sequelize\lib\sequelize.js:580:23)
    at databaseService.find (D:\AllCode\svntest\app\service\database.js:11:45)
    at databaseController.index (D:\AllCode\svntest\app\controller\database.js:32:51)
    at Object.callFn (D:\AllCode\svntest\node_modules\egg-core\lib\utils\index.js:44:21)
    at Object.classControllerMiddleware (D:\AllCode\svntest\node_modules\egg-core\lib\loader\mixin\controller.js:87:20)
    at Object.callFn (D:\AllCode\svntest\node_modules\@eggjs\router\lib\utils.js:12:21)
    at wrappedController (D:\AllCode\svntest\node_modules\@eggjs\router\lib\egg_router.js:322:18)
    at dispatch (D:\AllCode\svntest\node_modules\koa-compose\index.js:44:32)
    at next (D:\AllCode\svntest\node_modules\koa-compose\index.js:45:18)
    at D:\AllCode\svntest\node_modules\@eggjs\router\lib\router.js:190:18
    at dispatch (D:\AllCode\svntest\node_modules\koa-compose\index.js:44:32)
    at D:\AllCode\svntest\node_modules\koa-compose\index.js:36:12
    at dispatch (D:\AllCode\svntest\node_modules\@eggjs\router\lib\router.js:195:33)
    at dispatch (D:\AllCode\svntest\node_modules\koa\node_modules\koa-compose\index.js:42:32)
    at D:\AllCode\svntest\node_modules\egg-development\app\middleware\egg_loader_trace.js:9:56
    at dispatch (D:\AllCode\svntest\node_modules\koa\node_modules\koa-compose\index.js:42:32)
    at i18n (D:\AllCode\svntest\node_modules\egg-i18n\app\middleware\i18n.js:13:12)
    at dispatch (D:\AllCode\svntest\node_modules\koa\node_modules\koa-compose\index.js:42:32)
    at dispatch (D:\AllCode\svntest\node_modules\egg-security\node_modules\koa-compose\index.js:42:32)
    at dta (D:\AllCode\svntest\node_modules\egg-security\lib\middlewares\dta.js:12:12)
    at dispatch (D:\AllCode\svntest\node_modules\egg-security\node_modules\koa-compose\index.js:42:32)
    at xframe (D:\AllCode\svntest\node_modules\egg-security\lib\middlewares\xframe.js:7:11)
    at dispatch (D:\AllCode\svntest\node_modules\egg-security\node_modules\koa-compose\index.js:42:32)
    at xssProtection (D:\AllCode\svntest\node_modules\egg-security\lib\middlewares\xssProtection.js:7:11)
    at dispatch (D:\AllCode\svntest\node_modules\egg-security\node_modules\koa-compose\index.js:42:32)
    at nosniff (D:\AllCode\svntest\node_modules\egg-security\lib\middlewares\nosniff.js:8:11)
name: "SequelizeConnectionRefusedError"
parent: {"errno":"ECONNREFUSED","code":"ECONNREFUSED","syscall":"connect","address":"127.0.0.1","port":3306,"fatal":true}
original: {"errno":"ECONNREFUSED","code":"ECONNREFUSED","syscall":"connect","address":"127.0.0.1","port":3306,"fatal":true}
headers: {"Access-Control-Allow-Origin":"http://127.0.0.1:8080","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 5336
hostname: SC-201804270139

2019-09-30 13:40:34,594 ERROR 5236 [-/192.168.17.149/-/11ms GET /database/find] nodejs.TypeError: Cannot read property 'database' of undefined
    at databaseController.index (D:\AllCode\svntest\app\controller\database.js:28:46)
    at Object.callFn (D:\AllCode\svntest\node_modules\egg-core\lib\utils\index.js:44:21)
    at Object.classControllerMiddleware (D:\AllCode\svntest\node_modules\egg-core\lib\loader\mixin\controller.js:87:20)
    at Object.callFn (D:\AllCode\svntest\node_modules\@eggjs\router\lib\utils.js:12:21)
    at wrappedController (D:\AllCode\svntest\node_modules\@eggjs\router\lib\egg_router.js:322:18)
    at dispatch (D:\AllCode\svntest\node_modules\koa-compose\index.js:44:32)
    at next (D:\AllCode\svntest\node_modules\koa-compose\index.js:45:18)
    at D:\AllCode\svntest\node_modules\@eggjs\router\lib\router.js:190:18
    at dispatch (D:\AllCode\svntest\node_modules\koa-compose\index.js:44:32)
    at D:\AllCode\svntest\node_modules\koa-compose\index.js:36:12
    at dispatch (D:\AllCode\svntest\node_modules\@eggjs\router\lib\router.js:195:33)
    at dispatch (D:\AllCode\svntest\node_modules\koa\node_modules\koa-compose\index.js:42:32)
    at D:\AllCode\svntest\node_modules\egg-development\app\middleware\egg_loader_trace.js:9:56
    at dispatch (D:\AllCode\svntest\node_modules\koa\node_modules\koa-compose\index.js:42:32)
    at i18n (D:\AllCode\svntest\node_modules\egg-i18n\app\middleware\i18n.js:13:12)
    at dispatch (D:\AllCode\svntest\node_modules\koa\node_modules\koa-compose\index.js:42:32)
    at dispatch (D:\AllCode\svntest\node_modules\egg-security\node_modules\koa-compose\index.js:42:32)
    at dta (D:\AllCode\svntest\node_modules\egg-security\lib\middlewares\dta.js:12:12)
    at dispatch (D:\AllCode\svntest\node_modules\egg-security\node_modules\koa-compose\index.js:42:32)
    at xframe (D:\AllCode\svntest\node_modules\egg-security\lib\middlewares\xframe.js:7:11)
    at dispatch (D:\AllCode\svntest\node_modules\egg-security\node_modules\koa-compose\index.js:42:32)
    at xssProtection (D:\AllCode\svntest\node_modules\egg-security\lib\middlewares\xssProtection.js:7:11)
headers: {"Access-Control-Allow-Origin":"http://127.0.0.1:8080","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 5236
hostname: SC-201804270139

2019-09-30 13:40:39,739 ERROR 5236 [-/192.168.17.149/-/10ms GET /database/find] nodejs.TypeError: Cannot read property 'database' of undefined
    at databaseController.index (D:\AllCode\svntest\app\controller\database.js:28:46)
    at Object.callFn (D:\AllCode\svntest\node_modules\egg-core\lib\utils\index.js:44:21)
    at Object.classControllerMiddleware (D:\AllCode\svntest\node_modules\egg-core\lib\loader\mixin\controller.js:87:20)
    at Object.callFn (D:\AllCode\svntest\node_modules\@eggjs\router\lib\utils.js:12:21)
    at wrappedController (D:\AllCode\svntest\node_modules\@eggjs\router\lib\egg_router.js:322:18)
    at dispatch (D:\AllCode\svntest\node_modules\koa-compose\index.js:44:32)
    at next (D:\AllCode\svntest\node_modules\koa-compose\index.js:45:18)
    at D:\AllCode\svntest\node_modules\@eggjs\router\lib\router.js:190:18
    at dispatch (D:\AllCode\svntest\node_modules\koa-compose\index.js:44:32)
    at D:\AllCode\svntest\node_modules\koa-compose\index.js:36:12
    at dispatch (D:\AllCode\svntest\node_modules\@eggjs\router\lib\router.js:195:33)
    at dispatch (D:\AllCode\svntest\node_modules\koa\node_modules\koa-compose\index.js:42:32)
    at D:\AllCode\svntest\node_modules\egg-development\app\middleware\egg_loader_trace.js:9:56
    at dispatch (D:\AllCode\svntest\node_modules\koa\node_modules\koa-compose\index.js:42:32)
    at i18n (D:\AllCode\svntest\node_modules\egg-i18n\app\middleware\i18n.js:13:12)
    at dispatch (D:\AllCode\svntest\node_modules\koa\node_modules\koa-compose\index.js:42:32)
    at dispatch (D:\AllCode\svntest\node_modules\egg-security\node_modules\koa-compose\index.js:42:32)
    at dta (D:\AllCode\svntest\node_modules\egg-security\lib\middlewares\dta.js:12:12)
    at dispatch (D:\AllCode\svntest\node_modules\egg-security\node_modules\koa-compose\index.js:42:32)
    at xframe (D:\AllCode\svntest\node_modules\egg-security\lib\middlewares\xframe.js:7:11)
    at dispatch (D:\AllCode\svntest\node_modules\egg-security\node_modules\koa-compose\index.js:42:32)
    at xssProtection (D:\AllCode\svntest\node_modules\egg-security\lib\middlewares\xssProtection.js:7:11)
headers: {"Access-Control-Allow-Origin":"http://127.0.0.1:8080","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 5236
hostname: SC-201804270139

2019-09-30 13:41:13,221 ERROR 5236 [-/192.168.17.149/-/3ms GET /database/find] nodejs.TypeError: Cannot read property 'database' of undefined
    at databaseController.index (D:\AllCode\svntest\app\controller\database.js:28:46)
    at Object.callFn (D:\AllCode\svntest\node_modules\egg-core\lib\utils\index.js:44:21)
    at Object.classControllerMiddleware (D:\AllCode\svntest\node_modules\egg-core\lib\loader\mixin\controller.js:87:20)
    at Object.callFn (D:\AllCode\svntest\node_modules\@eggjs\router\lib\utils.js:12:21)
    at wrappedController (D:\AllCode\svntest\node_modules\@eggjs\router\lib\egg_router.js:322:18)
    at dispatch (D:\AllCode\svntest\node_modules\koa-compose\index.js:44:32)
    at next (D:\AllCode\svntest\node_modules\koa-compose\index.js:45:18)
    at D:\AllCode\svntest\node_modules\@eggjs\router\lib\router.js:190:18
    at dispatch (D:\AllCode\svntest\node_modules\koa-compose\index.js:44:32)
    at D:\AllCode\svntest\node_modules\koa-compose\index.js:36:12
    at dispatch (D:\AllCode\svntest\node_modules\@eggjs\router\lib\router.js:195:33)
    at dispatch (D:\AllCode\svntest\node_modules\koa\node_modules\koa-compose\index.js:42:32)
    at D:\AllCode\svntest\node_modules\egg-development\app\middleware\egg_loader_trace.js:9:56
    at dispatch (D:\AllCode\svntest\node_modules\koa\node_modules\koa-compose\index.js:42:32)
    at i18n (D:\AllCode\svntest\node_modules\egg-i18n\app\middleware\i18n.js:13:12)
    at dispatch (D:\AllCode\svntest\node_modules\koa\node_modules\koa-compose\index.js:42:32)
    at dispatch (D:\AllCode\svntest\node_modules\egg-security\node_modules\koa-compose\index.js:42:32)
    at dta (D:\AllCode\svntest\node_modules\egg-security\lib\middlewares\dta.js:12:12)
    at dispatch (D:\AllCode\svntest\node_modules\egg-security\node_modules\koa-compose\index.js:42:32)
    at xframe (D:\AllCode\svntest\node_modules\egg-security\lib\middlewares\xframe.js:7:11)
    at dispatch (D:\AllCode\svntest\node_modules\egg-security\node_modules\koa-compose\index.js:42:32)
    at xssProtection (D:\AllCode\svntest\node_modules\egg-security\lib\middlewares\xssProtection.js:7:11)
headers: {"Access-Control-Allow-Origin":"http://127.0.0.1:8080","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 5236
hostname: SC-201804270139

2019-09-30 13:41:17,050 ERROR 5236 [-/192.168.17.149/-/7ms GET /database/find] nodejs.TypeError: Cannot read property 'database' of undefined
    at databaseController.index (D:\AllCode\svntest\app\controller\database.js:28:46)
    at Object.callFn (D:\AllCode\svntest\node_modules\egg-core\lib\utils\index.js:44:21)
    at Object.classControllerMiddleware (D:\AllCode\svntest\node_modules\egg-core\lib\loader\mixin\controller.js:87:20)
    at Object.callFn (D:\AllCode\svntest\node_modules\@eggjs\router\lib\utils.js:12:21)
    at wrappedController (D:\AllCode\svntest\node_modules\@eggjs\router\lib\egg_router.js:322:18)
    at dispatch (D:\AllCode\svntest\node_modules\koa-compose\index.js:44:32)
    at next (D:\AllCode\svntest\node_modules\koa-compose\index.js:45:18)
    at D:\AllCode\svntest\node_modules\@eggjs\router\lib\router.js:190:18
    at dispatch (D:\AllCode\svntest\node_modules\koa-compose\index.js:44:32)
    at D:\AllCode\svntest\node_modules\koa-compose\index.js:36:12
    at dispatch (D:\AllCode\svntest\node_modules\@eggjs\router\lib\router.js:195:33)
    at dispatch (D:\AllCode\svntest\node_modules\koa\node_modules\koa-compose\index.js:42:32)
    at D:\AllCode\svntest\node_modules\egg-development\app\middleware\egg_loader_trace.js:9:56
    at dispatch (D:\AllCode\svntest\node_modules\koa\node_modules\koa-compose\index.js:42:32)
    at i18n (D:\AllCode\svntest\node_modules\egg-i18n\app\middleware\i18n.js:13:12)
    at dispatch (D:\AllCode\svntest\node_modules\koa\node_modules\koa-compose\index.js:42:32)
    at dispatch (D:\AllCode\svntest\node_modules\egg-security\node_modules\koa-compose\index.js:42:32)
    at dta (D:\AllCode\svntest\node_modules\egg-security\lib\middlewares\dta.js:12:12)
    at dispatch (D:\AllCode\svntest\node_modules\egg-security\node_modules\koa-compose\index.js:42:32)
    at xframe (D:\AllCode\svntest\node_modules\egg-security\lib\middlewares\xframe.js:7:11)
    at dispatch (D:\AllCode\svntest\node_modules\egg-security\node_modules\koa-compose\index.js:42:32)
    at xssProtection (D:\AllCode\svntest\node_modules\egg-security\lib\middlewares\xssProtection.js:7:11)
headers: {"Access-Control-Allow-Origin":"http://127.0.0.1:8080","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 5236
hostname: SC-201804270139

2019-09-30 13:41:41,593 ERROR 5236 [-/192.168.17.149/-/15ms GET /database/find] nodejs.TypeError: Cannot read property 'database' of undefined
    at databaseController.index (D:\AllCode\svntest\app\controller\database.js:28:46)
    at Object.callFn (D:\AllCode\svntest\node_modules\egg-core\lib\utils\index.js:44:21)
    at Object.classControllerMiddleware (D:\AllCode\svntest\node_modules\egg-core\lib\loader\mixin\controller.js:87:20)
    at Object.callFn (D:\AllCode\svntest\node_modules\@eggjs\router\lib\utils.js:12:21)
    at wrappedController (D:\AllCode\svntest\node_modules\@eggjs\router\lib\egg_router.js:322:18)
    at dispatch (D:\AllCode\svntest\node_modules\koa-compose\index.js:44:32)
    at next (D:\AllCode\svntest\node_modules\koa-compose\index.js:45:18)
    at D:\AllCode\svntest\node_modules\@eggjs\router\lib\router.js:190:18
    at dispatch (D:\AllCode\svntest\node_modules\koa-compose\index.js:44:32)
    at D:\AllCode\svntest\node_modules\koa-compose\index.js:36:12
    at dispatch (D:\AllCode\svntest\node_modules\@eggjs\router\lib\router.js:195:33)
    at dispatch (D:\AllCode\svntest\node_modules\koa\node_modules\koa-compose\index.js:42:32)
    at D:\AllCode\svntest\node_modules\egg-development\app\middleware\egg_loader_trace.js:9:56
    at dispatch (D:\AllCode\svntest\node_modules\koa\node_modules\koa-compose\index.js:42:32)
    at i18n (D:\AllCode\svntest\node_modules\egg-i18n\app\middleware\i18n.js:13:12)
    at dispatch (D:\AllCode\svntest\node_modules\koa\node_modules\koa-compose\index.js:42:32)
    at dispatch (D:\AllCode\svntest\node_modules\egg-security\node_modules\koa-compose\index.js:42:32)
    at dta (D:\AllCode\svntest\node_modules\egg-security\lib\middlewares\dta.js:12:12)
    at dispatch (D:\AllCode\svntest\node_modules\egg-security\node_modules\koa-compose\index.js:42:32)
    at xframe (D:\AllCode\svntest\node_modules\egg-security\lib\middlewares\xframe.js:7:11)
    at dispatch (D:\AllCode\svntest\node_modules\egg-security\node_modules\koa-compose\index.js:42:32)
    at xssProtection (D:\AllCode\svntest\node_modules\egg-security\lib\middlewares\xssProtection.js:7:11)
headers: {"Access-Control-Allow-Origin":"http://127.0.0.1:8080","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 5236
hostname: SC-201804270139

2019-09-30 13:41:45,409 ERROR 5236 [-/192.168.17.149/-/2ms GET /database/find] nodejs.TypeError: Cannot read property 'database' of undefined
    at databaseController.index (D:\AllCode\svntest\app\controller\database.js:28:46)
    at Object.callFn (D:\AllCode\svntest\node_modules\egg-core\lib\utils\index.js:44:21)
    at Object.classControllerMiddleware (D:\AllCode\svntest\node_modules\egg-core\lib\loader\mixin\controller.js:87:20)
    at Object.callFn (D:\AllCode\svntest\node_modules\@eggjs\router\lib\utils.js:12:21)
    at wrappedController (D:\AllCode\svntest\node_modules\@eggjs\router\lib\egg_router.js:322:18)
    at dispatch (D:\AllCode\svntest\node_modules\koa-compose\index.js:44:32)
    at next (D:\AllCode\svntest\node_modules\koa-compose\index.js:45:18)
    at D:\AllCode\svntest\node_modules\@eggjs\router\lib\router.js:190:18
    at dispatch (D:\AllCode\svntest\node_modules\koa-compose\index.js:44:32)
    at D:\AllCode\svntest\node_modules\koa-compose\index.js:36:12
    at dispatch (D:\AllCode\svntest\node_modules\@eggjs\router\lib\router.js:195:33)
    at dispatch (D:\AllCode\svntest\node_modules\koa\node_modules\koa-compose\index.js:42:32)
    at D:\AllCode\svntest\node_modules\egg-development\app\middleware\egg_loader_trace.js:9:56
    at dispatch (D:\AllCode\svntest\node_modules\koa\node_modules\koa-compose\index.js:42:32)
    at i18n (D:\AllCode\svntest\node_modules\egg-i18n\app\middleware\i18n.js:13:12)
    at dispatch (D:\AllCode\svntest\node_modules\koa\node_modules\koa-compose\index.js:42:32)
    at dispatch (D:\AllCode\svntest\node_modules\egg-security\node_modules\koa-compose\index.js:42:32)
    at dta (D:\AllCode\svntest\node_modules\egg-security\lib\middlewares\dta.js:12:12)
    at dispatch (D:\AllCode\svntest\node_modules\egg-security\node_modules\koa-compose\index.js:42:32)
    at xframe (D:\AllCode\svntest\node_modules\egg-security\lib\middlewares\xframe.js:7:11)
    at dispatch (D:\AllCode\svntest\node_modules\egg-security\node_modules\koa-compose\index.js:42:32)
    at xssProtection (D:\AllCode\svntest\node_modules\egg-security\lib\middlewares\xssProtection.js:7:11)
headers: {"Access-Control-Allow-Origin":"http://127.0.0.1:8080","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 5236
hostname: SC-201804270139

2019-09-30 13:42:08,536 ERROR 5236 [-/192.168.17.149/-/9ms GET /database/find] nodejs.TypeError: Cannot read property 'database' of undefined
    at databaseController.index (D:\AllCode\svntest\app\controller\database.js:28:46)
    at Object.callFn (D:\AllCode\svntest\node_modules\egg-core\lib\utils\index.js:44:21)
    at Object.classControllerMiddleware (D:\AllCode\svntest\node_modules\egg-core\lib\loader\mixin\controller.js:87:20)
    at Object.callFn (D:\AllCode\svntest\node_modules\@eggjs\router\lib\utils.js:12:21)
    at wrappedController (D:\AllCode\svntest\node_modules\@eggjs\router\lib\egg_router.js:322:18)
    at dispatch (D:\AllCode\svntest\node_modules\koa-compose\index.js:44:32)
    at next (D:\AllCode\svntest\node_modules\koa-compose\index.js:45:18)
    at D:\AllCode\svntest\node_modules\@eggjs\router\lib\router.js:190:18
    at dispatch (D:\AllCode\svntest\node_modules\koa-compose\index.js:44:32)
    at D:\AllCode\svntest\node_modules\koa-compose\index.js:36:12
    at dispatch (D:\AllCode\svntest\node_modules\@eggjs\router\lib\router.js:195:33)
    at dispatch (D:\AllCode\svntest\node_modules\koa\node_modules\koa-compose\index.js:42:32)
    at D:\AllCode\svntest\node_modules\egg-development\app\middleware\egg_loader_trace.js:9:56
    at dispatch (D:\AllCode\svntest\node_modules\koa\node_modules\koa-compose\index.js:42:32)
    at i18n (D:\AllCode\svntest\node_modules\egg-i18n\app\middleware\i18n.js:13:12)
    at dispatch (D:\AllCode\svntest\node_modules\koa\node_modules\koa-compose\index.js:42:32)
    at dispatch (D:\AllCode\svntest\node_modules\egg-security\node_modules\koa-compose\index.js:42:32)
    at dta (D:\AllCode\svntest\node_modules\egg-security\lib\middlewares\dta.js:12:12)
    at dispatch (D:\AllCode\svntest\node_modules\egg-security\node_modules\koa-compose\index.js:42:32)
    at xframe (D:\AllCode\svntest\node_modules\egg-security\lib\middlewares\xframe.js:7:11)
    at dispatch (D:\AllCode\svntest\node_modules\egg-security\node_modules\koa-compose\index.js:42:32)
    at xssProtection (D:\AllCode\svntest\node_modules\egg-security\lib\middlewares\xssProtection.js:7:11)
headers: {"Access-Control-Allow-Origin":"http://127.0.0.1:8080","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 5236
hostname: SC-201804270139

2019-09-30 13:42:12,974 ERROR 5236 [-/192.168.17.149/-/3ms GET /database/find] nodejs.TypeError: Cannot read property 'database' of undefined
    at databaseController.index (D:\AllCode\svntest\app\controller\database.js:28:46)
    at Object.callFn (D:\AllCode\svntest\node_modules\egg-core\lib\utils\index.js:44:21)
    at Object.classControllerMiddleware (D:\AllCode\svntest\node_modules\egg-core\lib\loader\mixin\controller.js:87:20)
    at Object.callFn (D:\AllCode\svntest\node_modules\@eggjs\router\lib\utils.js:12:21)
    at wrappedController (D:\AllCode\svntest\node_modules\@eggjs\router\lib\egg_router.js:322:18)
    at dispatch (D:\AllCode\svntest\node_modules\koa-compose\index.js:44:32)
    at next (D:\AllCode\svntest\node_modules\koa-compose\index.js:45:18)
    at D:\AllCode\svntest\node_modules\@eggjs\router\lib\router.js:190:18
    at dispatch (D:\AllCode\svntest\node_modules\koa-compose\index.js:44:32)
    at D:\AllCode\svntest\node_modules\koa-compose\index.js:36:12
    at dispatch (D:\AllCode\svntest\node_modules\@eggjs\router\lib\router.js:195:33)
    at dispatch (D:\AllCode\svntest\node_modules\koa\node_modules\koa-compose\index.js:42:32)
    at D:\AllCode\svntest\node_modules\egg-development\app\middleware\egg_loader_trace.js:9:56
    at dispatch (D:\AllCode\svntest\node_modules\koa\node_modules\koa-compose\index.js:42:32)
    at i18n (D:\AllCode\svntest\node_modules\egg-i18n\app\middleware\i18n.js:13:12)
    at dispatch (D:\AllCode\svntest\node_modules\koa\node_modules\koa-compose\index.js:42:32)
    at dispatch (D:\AllCode\svntest\node_modules\egg-security\node_modules\koa-compose\index.js:42:32)
    at dta (D:\AllCode\svntest\node_modules\egg-security\lib\middlewares\dta.js:12:12)
    at dispatch (D:\AllCode\svntest\node_modules\egg-security\node_modules\koa-compose\index.js:42:32)
    at xframe (D:\AllCode\svntest\node_modules\egg-security\lib\middlewares\xframe.js:7:11)
    at dispatch (D:\AllCode\svntest\node_modules\egg-security\node_modules\koa-compose\index.js:42:32)
    at xssProtection (D:\AllCode\svntest\node_modules\egg-security\lib\middlewares\xssProtection.js:7:11)
headers: {"Access-Control-Allow-Origin":"http://127.0.0.1:8080","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 5236
hostname: SC-201804270139

2019-09-30 13:42:38,168 ERROR 5236 [-/192.168.17.149/-/2ms GET /database/find] nodejs.TypeError: Cannot read property 'database' of undefined
    at databaseController.index (D:\AllCode\svntest\app\controller\database.js:28:46)
    at Object.callFn (D:\AllCode\svntest\node_modules\egg-core\lib\utils\index.js:44:21)
    at Object.classControllerMiddleware (D:\AllCode\svntest\node_modules\egg-core\lib\loader\mixin\controller.js:87:20)
    at Object.callFn (D:\AllCode\svntest\node_modules\@eggjs\router\lib\utils.js:12:21)
    at wrappedController (D:\AllCode\svntest\node_modules\@eggjs\router\lib\egg_router.js:322:18)
    at dispatch (D:\AllCode\svntest\node_modules\koa-compose\index.js:44:32)
    at next (D:\AllCode\svntest\node_modules\koa-compose\index.js:45:18)
    at D:\AllCode\svntest\node_modules\@eggjs\router\lib\router.js:190:18
    at dispatch (D:\AllCode\svntest\node_modules\koa-compose\index.js:44:32)
    at D:\AllCode\svntest\node_modules\koa-compose\index.js:36:12
    at dispatch (D:\AllCode\svntest\node_modules\@eggjs\router\lib\router.js:195:33)
    at dispatch (D:\AllCode\svntest\node_modules\koa\node_modules\koa-compose\index.js:42:32)
    at D:\AllCode\svntest\node_modules\egg-development\app\middleware\egg_loader_trace.js:9:56
    at dispatch (D:\AllCode\svntest\node_modules\koa\node_modules\koa-compose\index.js:42:32)
    at i18n (D:\AllCode\svntest\node_modules\egg-i18n\app\middleware\i18n.js:13:12)
    at dispatch (D:\AllCode\svntest\node_modules\koa\node_modules\koa-compose\index.js:42:32)
    at dispatch (D:\AllCode\svntest\node_modules\egg-security\node_modules\koa-compose\index.js:42:32)
    at dta (D:\AllCode\svntest\node_modules\egg-security\lib\middlewares\dta.js:12:12)
    at dispatch (D:\AllCode\svntest\node_modules\egg-security\node_modules\koa-compose\index.js:42:32)
    at xframe (D:\AllCode\svntest\node_modules\egg-security\lib\middlewares\xframe.js:7:11)
    at dispatch (D:\AllCode\svntest\node_modules\egg-security\node_modules\koa-compose\index.js:42:32)
    at xssProtection (D:\AllCode\svntest\node_modules\egg-security\lib\middlewares\xssProtection.js:7:11)
headers: {"Access-Control-Allow-Origin":"http://127.0.0.1:8080","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 5236
hostname: SC-201804270139

2019-09-30 13:42:42,873 ERROR 5236 [-/192.168.17.149/-/5ms GET /database/find] nodejs.TypeError: Cannot read property 'database' of undefined
    at databaseController.index (D:\AllCode\svntest\app\controller\database.js:28:46)
    at Object.callFn (D:\AllCode\svntest\node_modules\egg-core\lib\utils\index.js:44:21)
    at Object.classControllerMiddleware (D:\AllCode\svntest\node_modules\egg-core\lib\loader\mixin\controller.js:87:20)
    at Object.callFn (D:\AllCode\svntest\node_modules\@eggjs\router\lib\utils.js:12:21)
    at wrappedController (D:\AllCode\svntest\node_modules\@eggjs\router\lib\egg_router.js:322:18)
    at dispatch (D:\AllCode\svntest\node_modules\koa-compose\index.js:44:32)
    at next (D:\AllCode\svntest\node_modules\koa-compose\index.js:45:18)
    at D:\AllCode\svntest\node_modules\@eggjs\router\lib\router.js:190:18
    at dispatch (D:\AllCode\svntest\node_modules\koa-compose\index.js:44:32)
    at D:\AllCode\svntest\node_modules\koa-compose\index.js:36:12
    at dispatch (D:\AllCode\svntest\node_modules\@eggjs\router\lib\router.js:195:33)
    at dispatch (D:\AllCode\svntest\node_modules\koa\node_modules\koa-compose\index.js:42:32)
    at D:\AllCode\svntest\node_modules\egg-development\app\middleware\egg_loader_trace.js:9:56
    at dispatch (D:\AllCode\svntest\node_modules\koa\node_modules\koa-compose\index.js:42:32)
    at i18n (D:\AllCode\svntest\node_modules\egg-i18n\app\middleware\i18n.js:13:12)
    at dispatch (D:\AllCode\svntest\node_modules\koa\node_modules\koa-compose\index.js:42:32)
    at dispatch (D:\AllCode\svntest\node_modules\egg-security\node_modules\koa-compose\index.js:42:32)
    at dta (D:\AllCode\svntest\node_modules\egg-security\lib\middlewares\dta.js:12:12)
    at dispatch (D:\AllCode\svntest\node_modules\egg-security\node_modules\koa-compose\index.js:42:32)
    at xframe (D:\AllCode\svntest\node_modules\egg-security\lib\middlewares\xframe.js:7:11)
    at dispatch (D:\AllCode\svntest\node_modules\egg-security\node_modules\koa-compose\index.js:42:32)
    at xssProtection (D:\AllCode\svntest\node_modules\egg-security\lib\middlewares\xssProtection.js:7:11)
headers: {"Access-Control-Allow-Origin":"http://127.0.0.1:8080","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 5236
hostname: SC-201804270139

2019-09-30 13:42:59,413 ERROR 5236 [-/192.168.17.149/-/2ms GET /database/find] nodejs.TypeError: Cannot read property 'database' of undefined
    at databaseController.index (D:\AllCode\svntest\app\controller\database.js:28:46)
    at Object.callFn (D:\AllCode\svntest\node_modules\egg-core\lib\utils\index.js:44:21)
    at Object.classControllerMiddleware (D:\AllCode\svntest\node_modules\egg-core\lib\loader\mixin\controller.js:87:20)
    at Object.callFn (D:\AllCode\svntest\node_modules\@eggjs\router\lib\utils.js:12:21)
    at wrappedController (D:\AllCode\svntest\node_modules\@eggjs\router\lib\egg_router.js:322:18)
    at dispatch (D:\AllCode\svntest\node_modules\koa-compose\index.js:44:32)
    at next (D:\AllCode\svntest\node_modules\koa-compose\index.js:45:18)
    at D:\AllCode\svntest\node_modules\@eggjs\router\lib\router.js:190:18
    at dispatch (D:\AllCode\svntest\node_modules\koa-compose\index.js:44:32)
    at D:\AllCode\svntest\node_modules\koa-compose\index.js:36:12
    at dispatch (D:\AllCode\svntest\node_modules\@eggjs\router\lib\router.js:195:33)
    at dispatch (D:\AllCode\svntest\node_modules\koa\node_modules\koa-compose\index.js:42:32)
    at D:\AllCode\svntest\node_modules\egg-development\app\middleware\egg_loader_trace.js:9:56
    at dispatch (D:\AllCode\svntest\node_modules\koa\node_modules\koa-compose\index.js:42:32)
    at i18n (D:\AllCode\svntest\node_modules\egg-i18n\app\middleware\i18n.js:13:12)
    at dispatch (D:\AllCode\svntest\node_modules\koa\node_modules\koa-compose\index.js:42:32)
    at dispatch (D:\AllCode\svntest\node_modules\egg-security\node_modules\koa-compose\index.js:42:32)
    at dta (D:\AllCode\svntest\node_modules\egg-security\lib\middlewares\dta.js:12:12)
    at dispatch (D:\AllCode\svntest\node_modules\egg-security\node_modules\koa-compose\index.js:42:32)
    at xframe (D:\AllCode\svntest\node_modules\egg-security\lib\middlewares\xframe.js:7:11)
    at dispatch (D:\AllCode\svntest\node_modules\egg-security\node_modules\koa-compose\index.js:42:32)
    at xssProtection (D:\AllCode\svntest\node_modules\egg-security\lib\middlewares\xssProtection.js:7:11)
headers: {"Access-Control-Allow-Origin":"http://127.0.0.1:8080","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 5236
hostname: SC-201804270139

2019-09-30 13:43:05,317 ERROR 5236 [-/192.168.17.149/-/1ms GET /database/find] nodejs.TypeError: Cannot read property 'database' of undefined
    at databaseController.index (D:\AllCode\svntest\app\controller\database.js:28:46)
    at Object.callFn (D:\AllCode\svntest\node_modules\egg-core\lib\utils\index.js:44:21)
    at Object.classControllerMiddleware (D:\AllCode\svntest\node_modules\egg-core\lib\loader\mixin\controller.js:87:20)
    at Object.callFn (D:\AllCode\svntest\node_modules\@eggjs\router\lib\utils.js:12:21)
    at wrappedController (D:\AllCode\svntest\node_modules\@eggjs\router\lib\egg_router.js:322:18)
    at dispatch (D:\AllCode\svntest\node_modules\koa-compose\index.js:44:32)
    at next (D:\AllCode\svntest\node_modules\koa-compose\index.js:45:18)
    at D:\AllCode\svntest\node_modules\@eggjs\router\lib\router.js:190:18
    at dispatch (D:\AllCode\svntest\node_modules\koa-compose\index.js:44:32)
    at D:\AllCode\svntest\node_modules\koa-compose\index.js:36:12
    at dispatch (D:\AllCode\svntest\node_modules\@eggjs\router\lib\router.js:195:33)
    at dispatch (D:\AllCode\svntest\node_modules\koa\node_modules\koa-compose\index.js:42:32)
    at D:\AllCode\svntest\node_modules\egg-development\app\middleware\egg_loader_trace.js:9:56
    at dispatch (D:\AllCode\svntest\node_modules\koa\node_modules\koa-compose\index.js:42:32)
    at i18n (D:\AllCode\svntest\node_modules\egg-i18n\app\middleware\i18n.js:13:12)
    at dispatch (D:\AllCode\svntest\node_modules\koa\node_modules\koa-compose\index.js:42:32)
    at dispatch (D:\AllCode\svntest\node_modules\egg-security\node_modules\koa-compose\index.js:42:32)
    at dta (D:\AllCode\svntest\node_modules\egg-security\lib\middlewares\dta.js:12:12)
    at dispatch (D:\AllCode\svntest\node_modules\egg-security\node_modules\koa-compose\index.js:42:32)
    at xframe (D:\AllCode\svntest\node_modules\egg-security\lib\middlewares\xframe.js:7:11)
    at dispatch (D:\AllCode\svntest\node_modules\egg-security\node_modules\koa-compose\index.js:42:32)
    at xssProtection (D:\AllCode\svntest\node_modules\egg-security\lib\middlewares\xssProtection.js:7:11)
headers: {"Access-Control-Allow-Origin":"http://127.0.0.1:8080","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 5236
hostname: SC-201804270139

2019-09-30 13:43:17,928 ERROR 5236 [-/192.168.17.149/-/5ms GET /database/find] nodejs.TypeError: Cannot read property 'database' of undefined
    at databaseController.index (D:\AllCode\svntest\app\controller\database.js:28:46)
    at Object.callFn (D:\AllCode\svntest\node_modules\egg-core\lib\utils\index.js:44:21)
    at Object.classControllerMiddleware (D:\AllCode\svntest\node_modules\egg-core\lib\loader\mixin\controller.js:87:20)
    at Object.callFn (D:\AllCode\svntest\node_modules\@eggjs\router\lib\utils.js:12:21)
    at wrappedController (D:\AllCode\svntest\node_modules\@eggjs\router\lib\egg_router.js:322:18)
    at dispatch (D:\AllCode\svntest\node_modules\koa-compose\index.js:44:32)
    at next (D:\AllCode\svntest\node_modules\koa-compose\index.js:45:18)
    at D:\AllCode\svntest\node_modules\@eggjs\router\lib\router.js:190:18
    at dispatch (D:\AllCode\svntest\node_modules\koa-compose\index.js:44:32)
    at D:\AllCode\svntest\node_modules\koa-compose\index.js:36:12
    at dispatch (D:\AllCode\svntest\node_modules\@eggjs\router\lib\router.js:195:33)
    at dispatch (D:\AllCode\svntest\node_modules\koa\node_modules\koa-compose\index.js:42:32)
    at D:\AllCode\svntest\node_modules\egg-development\app\middleware\egg_loader_trace.js:9:56
    at dispatch (D:\AllCode\svntest\node_modules\koa\node_modules\koa-compose\index.js:42:32)
    at i18n (D:\AllCode\svntest\node_modules\egg-i18n\app\middleware\i18n.js:13:12)
    at dispatch (D:\AllCode\svntest\node_modules\koa\node_modules\koa-compose\index.js:42:32)
    at dispatch (D:\AllCode\svntest\node_modules\egg-security\node_modules\koa-compose\index.js:42:32)
    at dta (D:\AllCode\svntest\node_modules\egg-security\lib\middlewares\dta.js:12:12)
    at dispatch (D:\AllCode\svntest\node_modules\egg-security\node_modules\koa-compose\index.js:42:32)
    at xframe (D:\AllCode\svntest\node_modules\egg-security\lib\middlewares\xframe.js:7:11)
    at dispatch (D:\AllCode\svntest\node_modules\egg-security\node_modules\koa-compose\index.js:42:32)
    at xssProtection (D:\AllCode\svntest\node_modules\egg-security\lib\middlewares\xssProtection.js:7:11)
headers: {"Access-Control-Allow-Origin":"http://127.0.0.1:8080","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 5236
hostname: SC-201804270139

2019-09-30 13:43:21,671 ERROR 5236 [-/192.168.17.149/-/10ms GET /database/find] nodejs.TypeError: Cannot read property 'database' of undefined
    at databaseController.index (D:\AllCode\svntest\app\controller\database.js:28:46)
    at Object.callFn (D:\AllCode\svntest\node_modules\egg-core\lib\utils\index.js:44:21)
    at Object.classControllerMiddleware (D:\AllCode\svntest\node_modules\egg-core\lib\loader\mixin\controller.js:87:20)
    at Object.callFn (D:\AllCode\svntest\node_modules\@eggjs\router\lib\utils.js:12:21)
    at wrappedController (D:\AllCode\svntest\node_modules\@eggjs\router\lib\egg_router.js:322:18)
    at dispatch (D:\AllCode\svntest\node_modules\koa-compose\index.js:44:32)
    at next (D:\AllCode\svntest\node_modules\koa-compose\index.js:45:18)
    at D:\AllCode\svntest\node_modules\@eggjs\router\lib\router.js:190:18
    at dispatch (D:\AllCode\svntest\node_modules\koa-compose\index.js:44:32)
    at D:\AllCode\svntest\node_modules\koa-compose\index.js:36:12
    at dispatch (D:\AllCode\svntest\node_modules\@eggjs\router\lib\router.js:195:33)
    at dispatch (D:\AllCode\svntest\node_modules\koa\node_modules\koa-compose\index.js:42:32)
    at D:\AllCode\svntest\node_modules\egg-development\app\middleware\egg_loader_trace.js:9:56
    at dispatch (D:\AllCode\svntest\node_modules\koa\node_modules\koa-compose\index.js:42:32)
    at i18n (D:\AllCode\svntest\node_modules\egg-i18n\app\middleware\i18n.js:13:12)
    at dispatch (D:\AllCode\svntest\node_modules\koa\node_modules\koa-compose\index.js:42:32)
    at dispatch (D:\AllCode\svntest\node_modules\egg-security\node_modules\koa-compose\index.js:42:32)
    at dta (D:\AllCode\svntest\node_modules\egg-security\lib\middlewares\dta.js:12:12)
    at dispatch (D:\AllCode\svntest\node_modules\egg-security\node_modules\koa-compose\index.js:42:32)
    at xframe (D:\AllCode\svntest\node_modules\egg-security\lib\middlewares\xframe.js:7:11)
    at dispatch (D:\AllCode\svntest\node_modules\egg-security\node_modules\koa-compose\index.js:42:32)
    at xssProtection (D:\AllCode\svntest\node_modules\egg-security\lib\middlewares\xssProtection.js:7:11)
headers: {"Access-Control-Allow-Origin":"http://127.0.0.1:8080","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 5236
hostname: SC-201804270139

2019-09-30 13:45:07,783 ERROR 5236 [-/192.168.17.149/-/2ms GET /database/find] nodejs.TypeError: Cannot read property 'database' of undefined
    at databaseController.index (D:\AllCode\svntest\app\controller\database.js:28:46)
    at Object.callFn (D:\AllCode\svntest\node_modules\egg-core\lib\utils\index.js:44:21)
    at Object.classControllerMiddleware (D:\AllCode\svntest\node_modules\egg-core\lib\loader\mixin\controller.js:87:20)
    at Object.callFn (D:\AllCode\svntest\node_modules\@eggjs\router\lib\utils.js:12:21)
    at wrappedController (D:\AllCode\svntest\node_modules\@eggjs\router\lib\egg_router.js:322:18)
    at dispatch (D:\AllCode\svntest\node_modules\koa-compose\index.js:44:32)
    at next (D:\AllCode\svntest\node_modules\koa-compose\index.js:45:18)
    at D:\AllCode\svntest\node_modules\@eggjs\router\lib\router.js:190:18
    at dispatch (D:\AllCode\svntest\node_modules\koa-compose\index.js:44:32)
    at D:\AllCode\svntest\node_modules\koa-compose\index.js:36:12
    at dispatch (D:\AllCode\svntest\node_modules\@eggjs\router\lib\router.js:195:33)
    at dispatch (D:\AllCode\svntest\node_modules\koa\node_modules\koa-compose\index.js:42:32)
    at D:\AllCode\svntest\node_modules\egg-development\app\middleware\egg_loader_trace.js:9:56
    at dispatch (D:\AllCode\svntest\node_modules\koa\node_modules\koa-compose\index.js:42:32)
    at i18n (D:\AllCode\svntest\node_modules\egg-i18n\app\middleware\i18n.js:13:12)
    at dispatch (D:\AllCode\svntest\node_modules\koa\node_modules\koa-compose\index.js:42:32)
    at dispatch (D:\AllCode\svntest\node_modules\egg-security\node_modules\koa-compose\index.js:42:32)
    at dta (D:\AllCode\svntest\node_modules\egg-security\lib\middlewares\dta.js:12:12)
    at dispatch (D:\AllCode\svntest\node_modules\egg-security\node_modules\koa-compose\index.js:42:32)
    at xframe (D:\AllCode\svntest\node_modules\egg-security\lib\middlewares\xframe.js:7:11)
    at dispatch (D:\AllCode\svntest\node_modules\egg-security\node_modules\koa-compose\index.js:42:32)
    at xssProtection (D:\AllCode\svntest\node_modules\egg-security\lib\middlewares\xssProtection.js:7:11)
headers: {"Access-Control-Allow-Origin":"http://127.0.0.1:8080","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 5236
hostname: SC-201804270139

2019-09-30 13:45:12,370 ERROR 5236 [-/192.168.17.149/-/8ms GET /database/find] nodejs.TypeError: Cannot read property 'database' of undefined
    at databaseController.index (D:\AllCode\svntest\app\controller\database.js:28:46)
    at Object.callFn (D:\AllCode\svntest\node_modules\egg-core\lib\utils\index.js:44:21)
    at Object.classControllerMiddleware (D:\AllCode\svntest\node_modules\egg-core\lib\loader\mixin\controller.js:87:20)
    at Object.callFn (D:\AllCode\svntest\node_modules\@eggjs\router\lib\utils.js:12:21)
    at wrappedController (D:\AllCode\svntest\node_modules\@eggjs\router\lib\egg_router.js:322:18)
    at dispatch (D:\AllCode\svntest\node_modules\koa-compose\index.js:44:32)
    at next (D:\AllCode\svntest\node_modules\koa-compose\index.js:45:18)
    at D:\AllCode\svntest\node_modules\@eggjs\router\lib\router.js:190:18
    at dispatch (D:\AllCode\svntest\node_modules\koa-compose\index.js:44:32)
    at D:\AllCode\svntest\node_modules\koa-compose\index.js:36:12
    at dispatch (D:\AllCode\svntest\node_modules\@eggjs\router\lib\router.js:195:33)
    at dispatch (D:\AllCode\svntest\node_modules\koa\node_modules\koa-compose\index.js:42:32)
    at D:\AllCode\svntest\node_modules\egg-development\app\middleware\egg_loader_trace.js:9:56
    at dispatch (D:\AllCode\svntest\node_modules\koa\node_modules\koa-compose\index.js:42:32)
    at i18n (D:\AllCode\svntest\node_modules\egg-i18n\app\middleware\i18n.js:13:12)
    at dispatch (D:\AllCode\svntest\node_modules\koa\node_modules\koa-compose\index.js:42:32)
    at dispatch (D:\AllCode\svntest\node_modules\egg-security\node_modules\koa-compose\index.js:42:32)
    at dta (D:\AllCode\svntest\node_modules\egg-security\lib\middlewares\dta.js:12:12)
    at dispatch (D:\AllCode\svntest\node_modules\egg-security\node_modules\koa-compose\index.js:42:32)
    at xframe (D:\AllCode\svntest\node_modules\egg-security\lib\middlewares\xframe.js:7:11)
    at dispatch (D:\AllCode\svntest\node_modules\egg-security\node_modules\koa-compose\index.js:42:32)
    at xssProtection (D:\AllCode\svntest\node_modules\egg-security\lib\middlewares\xssProtection.js:7:11)
headers: {"Access-Control-Allow-Origin":"http://127.0.0.1:8080","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 5236
hostname: SC-201804270139

2019-09-30 13:45:29,491 ERROR 5236 [-/192.168.17.149/-/4ms GET /database/find] nodejs.TypeError: Cannot read property 'database' of undefined
    at databaseController.index (D:\AllCode\svntest\app\controller\database.js:28:46)
    at Object.callFn (D:\AllCode\svntest\node_modules\egg-core\lib\utils\index.js:44:21)
    at Object.classControllerMiddleware (D:\AllCode\svntest\node_modules\egg-core\lib\loader\mixin\controller.js:87:20)
    at Object.callFn (D:\AllCode\svntest\node_modules\@eggjs\router\lib\utils.js:12:21)
    at wrappedController (D:\AllCode\svntest\node_modules\@eggjs\router\lib\egg_router.js:322:18)
    at dispatch (D:\AllCode\svntest\node_modules\koa-compose\index.js:44:32)
    at next (D:\AllCode\svntest\node_modules\koa-compose\index.js:45:18)
    at D:\AllCode\svntest\node_modules\@eggjs\router\lib\router.js:190:18
    at dispatch (D:\AllCode\svntest\node_modules\koa-compose\index.js:44:32)
    at D:\AllCode\svntest\node_modules\koa-compose\index.js:36:12
    at dispatch (D:\AllCode\svntest\node_modules\@eggjs\router\lib\router.js:195:33)
    at dispatch (D:\AllCode\svntest\node_modules\koa\node_modules\koa-compose\index.js:42:32)
    at D:\AllCode\svntest\node_modules\egg-development\app\middleware\egg_loader_trace.js:9:56
    at dispatch (D:\AllCode\svntest\node_modules\koa\node_modules\koa-compose\index.js:42:32)
    at i18n (D:\AllCode\svntest\node_modules\egg-i18n\app\middleware\i18n.js:13:12)
    at dispatch (D:\AllCode\svntest\node_modules\koa\node_modules\koa-compose\index.js:42:32)
    at dispatch (D:\AllCode\svntest\node_modules\egg-security\node_modules\koa-compose\index.js:42:32)
    at dta (D:\AllCode\svntest\node_modules\egg-security\lib\middlewares\dta.js:12:12)
    at dispatch (D:\AllCode\svntest\node_modules\egg-security\node_modules\koa-compose\index.js:42:32)
    at xframe (D:\AllCode\svntest\node_modules\egg-security\lib\middlewares\xframe.js:7:11)
    at dispatch (D:\AllCode\svntest\node_modules\egg-security\node_modules\koa-compose\index.js:42:32)
    at xssProtection (D:\AllCode\svntest\node_modules\egg-security\lib\middlewares\xssProtection.js:7:11)
headers: {"Access-Control-Allow-Origin":"http://127.0.0.1:8080","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 5236
hostname: SC-201804270139

2019-09-30 13:45:33,422 ERROR 5236 [-/192.168.17.149/-/2ms GET /database/find] nodejs.TypeError: Cannot read property 'database' of undefined
    at databaseController.index (D:\AllCode\svntest\app\controller\database.js:28:46)
    at Object.callFn (D:\AllCode\svntest\node_modules\egg-core\lib\utils\index.js:44:21)
    at Object.classControllerMiddleware (D:\AllCode\svntest\node_modules\egg-core\lib\loader\mixin\controller.js:87:20)
    at Object.callFn (D:\AllCode\svntest\node_modules\@eggjs\router\lib\utils.js:12:21)
    at wrappedController (D:\AllCode\svntest\node_modules\@eggjs\router\lib\egg_router.js:322:18)
    at dispatch (D:\AllCode\svntest\node_modules\koa-compose\index.js:44:32)
    at next (D:\AllCode\svntest\node_modules\koa-compose\index.js:45:18)
    at D:\AllCode\svntest\node_modules\@eggjs\router\lib\router.js:190:18
    at dispatch (D:\AllCode\svntest\node_modules\koa-compose\index.js:44:32)
    at D:\AllCode\svntest\node_modules\koa-compose\index.js:36:12
    at dispatch (D:\AllCode\svntest\node_modules\@eggjs\router\lib\router.js:195:33)
    at dispatch (D:\AllCode\svntest\node_modules\koa\node_modules\koa-compose\index.js:42:32)
    at D:\AllCode\svntest\node_modules\egg-development\app\middleware\egg_loader_trace.js:9:56
    at dispatch (D:\AllCode\svntest\node_modules\koa\node_modules\koa-compose\index.js:42:32)
    at i18n (D:\AllCode\svntest\node_modules\egg-i18n\app\middleware\i18n.js:13:12)
    at dispatch (D:\AllCode\svntest\node_modules\koa\node_modules\koa-compose\index.js:42:32)
    at dispatch (D:\AllCode\svntest\node_modules\egg-security\node_modules\koa-compose\index.js:42:32)
    at dta (D:\AllCode\svntest\node_modules\egg-security\lib\middlewares\dta.js:12:12)
    at dispatch (D:\AllCode\svntest\node_modules\egg-security\node_modules\koa-compose\index.js:42:32)
    at xframe (D:\AllCode\svntest\node_modules\egg-security\lib\middlewares\xframe.js:7:11)
    at dispatch (D:\AllCode\svntest\node_modules\egg-security\node_modules\koa-compose\index.js:42:32)
    at xssProtection (D:\AllCode\svntest\node_modules\egg-security\lib\middlewares\xssProtection.js:7:11)
headers: {"Access-Control-Allow-Origin":"http://127.0.0.1:8080","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 5236
hostname: SC-201804270139

2019-09-30 13:46:09,077 ERROR 5236 [-/192.168.17.149/-/1ms GET /database/find?database=keyan&host=192.168.17.194&port=3306&username=root&password=Juzhiyuan123.] nodejs.TypeError: Cannot read property 'database' of undefined
    at databaseController.index (D:\AllCode\svntest\app\controller\database.js:28:46)
    at Object.callFn (D:\AllCode\svntest\node_modules\egg-core\lib\utils\index.js:44:21)
    at Object.classControllerMiddleware (D:\AllCode\svntest\node_modules\egg-core\lib\loader\mixin\controller.js:87:20)
    at Object.callFn (D:\AllCode\svntest\node_modules\@eggjs\router\lib\utils.js:12:21)
    at wrappedController (D:\AllCode\svntest\node_modules\@eggjs\router\lib\egg_router.js:322:18)
    at dispatch (D:\AllCode\svntest\node_modules\koa-compose\index.js:44:32)
    at next (D:\AllCode\svntest\node_modules\koa-compose\index.js:45:18)
    at D:\AllCode\svntest\node_modules\@eggjs\router\lib\router.js:190:18
    at dispatch (D:\AllCode\svntest\node_modules\koa-compose\index.js:44:32)
    at D:\AllCode\svntest\node_modules\koa-compose\index.js:36:12
    at dispatch (D:\AllCode\svntest\node_modules\@eggjs\router\lib\router.js:195:33)
    at dispatch (D:\AllCode\svntest\node_modules\koa\node_modules\koa-compose\index.js:42:32)
    at D:\AllCode\svntest\node_modules\egg-development\app\middleware\egg_loader_trace.js:9:56
    at dispatch (D:\AllCode\svntest\node_modules\koa\node_modules\koa-compose\index.js:42:32)
    at i18n (D:\AllCode\svntest\node_modules\egg-i18n\app\middleware\i18n.js:13:12)
    at dispatch (D:\AllCode\svntest\node_modules\koa\node_modules\koa-compose\index.js:42:32)
    at dispatch (D:\AllCode\svntest\node_modules\egg-security\node_modules\koa-compose\index.js:42:32)
    at dta (D:\AllCode\svntest\node_modules\egg-security\lib\middlewares\dta.js:12:12)
    at dispatch (D:\AllCode\svntest\node_modules\egg-security\node_modules\koa-compose\index.js:42:32)
    at xframe (D:\AllCode\svntest\node_modules\egg-security\lib\middlewares\xframe.js:7:11)
    at dispatch (D:\AllCode\svntest\node_modules\egg-security\node_modules\koa-compose\index.js:42:32)
    at xssProtection (D:\AllCode\svntest\node_modules\egg-security\lib\middlewares\xssProtection.js:7:11)
headers: {"Access-Control-Allow-Origin":"http://127.0.0.1:8080","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 5236
hostname: SC-201804270139

2019-09-30 13:46:16,631 ERROR 5236 [-/192.168.17.149/-/2ms GET /database/find?database=keyan&host=192.168.17.194&port=3306&username=root&password=Juzhiyuan123.] nodejs.TypeError: Cannot read property 'database' of undefined
    at databaseController.index (D:\AllCode\svntest\app\controller\database.js:28:46)
    at Object.callFn (D:\AllCode\svntest\node_modules\egg-core\lib\utils\index.js:44:21)
    at Object.classControllerMiddleware (D:\AllCode\svntest\node_modules\egg-core\lib\loader\mixin\controller.js:87:20)
    at Object.callFn (D:\AllCode\svntest\node_modules\@eggjs\router\lib\utils.js:12:21)
    at wrappedController (D:\AllCode\svntest\node_modules\@eggjs\router\lib\egg_router.js:322:18)
    at dispatch (D:\AllCode\svntest\node_modules\koa-compose\index.js:44:32)
    at next (D:\AllCode\svntest\node_modules\koa-compose\index.js:45:18)
    at D:\AllCode\svntest\node_modules\@eggjs\router\lib\router.js:190:18
    at dispatch (D:\AllCode\svntest\node_modules\koa-compose\index.js:44:32)
    at D:\AllCode\svntest\node_modules\koa-compose\index.js:36:12
    at dispatch (D:\AllCode\svntest\node_modules\@eggjs\router\lib\router.js:195:33)
    at dispatch (D:\AllCode\svntest\node_modules\koa\node_modules\koa-compose\index.js:42:32)
    at D:\AllCode\svntest\node_modules\egg-development\app\middleware\egg_loader_trace.js:9:56
    at dispatch (D:\AllCode\svntest\node_modules\koa\node_modules\koa-compose\index.js:42:32)
    at i18n (D:\AllCode\svntest\node_modules\egg-i18n\app\middleware\i18n.js:13:12)
    at dispatch (D:\AllCode\svntest\node_modules\koa\node_modules\koa-compose\index.js:42:32)
    at dispatch (D:\AllCode\svntest\node_modules\egg-security\node_modules\koa-compose\index.js:42:32)
    at dta (D:\AllCode\svntest\node_modules\egg-security\lib\middlewares\dta.js:12:12)
    at dispatch (D:\AllCode\svntest\node_modules\egg-security\node_modules\koa-compose\index.js:42:32)
    at xframe (D:\AllCode\svntest\node_modules\egg-security\lib\middlewares\xframe.js:7:11)
    at dispatch (D:\AllCode\svntest\node_modules\egg-security\node_modules\koa-compose\index.js:42:32)
    at xssProtection (D:\AllCode\svntest\node_modules\egg-security\lib\middlewares\xssProtection.js:7:11)
headers: {"Access-Control-Allow-Origin":"http://127.0.0.1:8080","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 5236
hostname: SC-201804270139

